src/index.html

```diff
@@ -3,11 +3,322 @@
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="gallery" id="gallery">
+        <div class="container gallery__container">
+          <div class="gallery__title title">Галерея</div>
+          <div class="swiper">
+            <!-- Additional required wrapper -->
+            <div class="swiper-wrapper">
+              <!-- Slides -->
+              <div class="swiper-slide">
+                <img class="swiper-img"
+                  src="images/pictures/gallery-1.svg"
+                  alt="gallery-1">
+              </div>
+              <div class="swiper-slide">
+                <img class="swiper-img"
+                  src="images/pictures/gallery-2.svg"
+                  alt="gallery-2">
+              </div>
+              <div class="swiper-slide">
+                <img class="swiper-img"
+                  src="images/pictures/gallery-1.svg"
+                  alt="gallery-1">
+              </div>
+              ...
+            </div>
+            <!-- If we need pagination -->
+            <div class="swiper-pagination"></div>
+
+            <!-- If we need navigation buttons -->
+
+            <!-- If we need scrollbar -->
+          </div>
+          <div class="gallery__grid">
+            <div class="grid__row">
+              <img
+                src="images/pictures/gallery-1280-1.svg"
+                alt="gallery-1"
+                class=" grid__img grid__img-1"
+                >
+              <img
+                src="images/pictures/gallery-1280-2.svg"
+                alt="gallery-2"
+                class="gird__img grid__img-2"
+                >
+            </div>
+            <div class="grid__row">
+              <img
+                src="images/pictures/gallery-1280-3.svg"
+                alt="gallery-3"
+                class="gird__img grid__img-3"
+                >
+              <img
+                src="images/pictures/gallery-1280-4.svg"
+                alt="gallery-41"
+                class="gird__img grid__img-4"
+                >
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="subscription">
+        <div class="container subscription__container">
+          <div class="subscription__title title">Підписка</div>
+          <div class="subscription__subtitle subtitle">Бути в курсі всіх актуальних подій та останніх новин.</div>
+          <form
+          onsubmit="this.reset(); return false;"
+          class="form"
+           id="form"
+          >
+            <div class="tickets__row  tickets__row--orange tickets">
+              <input
+              type="email"
+              class="tickets__text tickets__input"
+              placeholder="E-mail"
+              required
+              >
+              <button type="submit"
+               class="tickets__link-row form__button
+                tickets__link--orange"
+              >
```

Comment:
Need to fix it everywhere
![image](https://github.com/mate-academy/Museum/assets/64750958/7716ceda-6ef2-4fb3-a017-77d38eac7c6d)